harness/mock-search/prepare_data.py

```diff
@@ -14,10 +14,11 @@
 # that will be run as a single job.
 with open(DNA_PATH, 'r') as dna_file:
     pairs = []
-    for index, line in enumerate(dna_file):
+    for index, line in enumerate(dna_file.readlines()):
         line = line.strip()
-        if line.startswith('#'):
+        if line.startswith('#') or len(line) == 0:
             continue
+        pairs.append([line[0], line[1]])
```

Comment:
What's this for? Isn't this adding the first two characters of the line as a separate sequence?